domain/annotation/errors/errors.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package errors
+
+import (
+	"github.com/juju/errors"
+)
+
+const (
+	// AnnotationUnknwonKind is raised when the Kind of an ID provided to the annotations state
+	// layer is not recognized
+	AnnotationUnknownKind = errors.ConstError("unknown kind")
```

Comment:
Just `UnknownKind` is enough given its package location. Typo, "Unknwon" above too.